scripts/deploy-contracts.sh

```
@@ -0,0 +1,210 @@
+REPO_ROOT="$(cd "$SCRIPT_DIR/.." && pwd)"
+cd "$REPO_ROOT"
+
+# ── Load .env / .env.local if present (without overriding already-set vars) ──
+load_env_file() {
+  local file="$1"
+  [[ -f "$file" ]] || return 0
+  log "Loading env from $file"
+  # shellcheck disable=SC2046
+  set -a
+  # Filter comments + blank lines so `set -a` only sees real assignments.
+  source <(grep -E '^[A-Za-z_][A-Za-z0-9_]*=' "$file" || true)
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)